bundled/tool/install_dependencies.py

```diff
@@ -0,0 +1,31 @@
+import subprocess
+import sys
+from pathlib import Path
+
+def install_dependencies(libsPath):
+    try:
+        import fastapi
+        import orjson
+    except ImportError:
+        subprocess.check_call(
+            [
+                sys.executable,
+                "-m",
+                "pip",
+                "install",
+                "fastapi>=0.100.0,<0.200.0",
+                "pydantic>=2.0.0", # In case of FastAPI installs pydantic==1
+                "orjson>=3.9, <4.0",
```

Comment:
NIL: can we move this to something like a requirement file, I just think it's a maintainence burden to have dependencies scatter in both code and file.

Reply:
done.